### ISSUE TITLE
Cannot Reproduce Results from the Paper on the VisDA-2017 Dataset

### ISSUE THREAD

**Issue by ki3nd** [Wenlve-Zhou/VLP-UDA#4]
I ran the code on the VisDA-2017 dataset with the following settings:

{
  "model_name": "RN101",
  "fixmatch": false,
  "rst": false  
  "lr": 3e-09,
  "momentum": 0.9,
  "weight_decay": 0.0005,
  "multiple_lr_classifier": 10000.0,
  "lambda1": 0.25,
  "lambda2": 0.1,
  "lambda3": 0.025,
  "clf_loss": "cross_entropy",
  "scheduler": true,
  "lr_gamma": 0.0003,
  "lr_decay": 0.75,
  "num_class": 12,
  "max_iter": 10000,
}
The result accuracy I obtained is 82.9%, while the accuracy using CLIP-pretrained prediction is 84.4%. Could you please share the training configuration used for the VisDA-2017 dataset with a ResNet-101 backbone?

**Comment by Wenlve-Zhou** [comment]
Hello! The 84.4% in the paper is the zero-shot inference result of CLIP, which is independent of the provided hyperparameter settings. You can obtain the corresponding results by running the following code:

`python test.py --datasets visda  --data_dir data/visda --src_domain train --tgt_domain validation --model_name RN101 --clip`

**Comment by ki3nd** [comment]
@Wenlve-Zhou  I trained CLIP model (backbone RN101) using the settings from the paper on dataset VisDA-2017. The result was poor (82.9%), which is lower than the zero-shot inference result of CLIP (before training). 

`python main.py --config configs/visda.yaml --data_dir ... --src_domain train --tgt_domain validation --model_name RN101 --lr 3e-9 --multiple_lr_classifier 10000`

**Comment by Wenlve-Zhou** [comment]
I understand your question. You can increase the value of --multiple_lr_classifier. For example, set it to --multiple_lr_classifier 100000, and you will achieve corresponding performance gains.

**Comment by k1endn** [comment]
Thank you very much. I ran the code again using your suggestion and obtained the same result as reported in the paper.
